test/e2e/specs/specs-playwright/wp-calypso__coblocks-spec.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import {
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	PublishedPostPage,
+	ClickToTweetBlock,
+	PricingTableBlock,
+} from '@automattic/calypso-e2e';
+
+describe( DataHelper.createSuiteTitle( 'Gutenberg: CoBlocks', { parallel: false } ), function () {
+	describe( DataHelper.createSubsuiteTitle( 'Click to Tweet', { parallel: true } ), function () {
```

Comment:
Note the use of the new method here.

The alternative is also acceptable, in case reviewer(s) think this method is unnecessary:

`describe( 'Click to Tweet: @parallel') ...`